qmcpy/algorithms/function/__init__.py

```diff
@@ -1,69 +0,0 @@
-''' Originally developed in MATLAB by Fred Hickernell. Translated to python by Sou-Cheng T. Choi and Aleksei Sorokin '''
```

Comment:
Also, putting historical comments in the code is odd -- generally the code is just considered alive and the GitHub history records how it got to where it is.